### ISSUE TITLE
[Can't check website for a11y] Question: when blocking against scripts usage on the website configured?

### ISSUE THREAD

**Issue by OlgaTCK** [TheJaredWilcurt/UGUI-pa11y#57]
[Can't check website for a11y]. 
"Run" button grayed out and general report in HTML is unavailible.
The image a11y section also not possible to use: The console generates “[]” only instead of the code to paste in Koa11y to enable “Run” button and make image a11y check.

URL used: https://www.oracle.com/index.html
1. Copy the code for image accessibility check from Koa11y and paste it in the console of the opened URL. Press “Enter”
2. Console content: 3 user messages
   ```
   Line 1. “Console was cleared”,
   Line 2. [], 
   Line 3. The above code has been copied to your clipboard
   Line 4. Undefined
   ```

Can this be the reason? was in the console when pressed F12:
```
•	onMessage extension  extension.js: 24
avast! Online Security plugin (c) 2013 Avast Corp.
@author: Lucian Corlaciu
Injected specifics - Google Chrome
•	Clean the cache of the scraper (new onComplete event)    ial.js:425
{
            console.log("Clean the cache of the scraper (new onComplete event)");
            AvastWRC.ial.sp.checkSafeShopDataConst = JSON.parse(JSON.stringify({
                data: null,
                completed: false,
                result: [],
                rescan: -1,
            }));
        },
```

**Comment by TheJaredWilcurt** [comment]
Working as expected. the script only looks for `<img>` tags, then finds their `src` and `alt` attribute values. There are no `<img>`'s on the page. all pictures shown are background images on `div`s. This is not accessible if you want visually impaired to be imformed of what the images look like. If these are decorative and not substantial, then describing the background images is not needed.

Running Koa11y 3 on that page returns 29 errors, 60 warnings, and 216 notices (WCAG 2.0 AA). If the RUN button is disabled:

* You need a URL/Destination/Filename and at least one button pressed (err/warn/notice), or some valid JSON in the imageAlts box
